rest-api/api/awardee_api.py

```diff
@@ -0,0 +1,19 @@
+from api.base_api import BaseApi
+from app_util import auth_required
+from api_util import PTC_AND_HEALTHPRO
+from dao.hpo_dao import HPODao
+from werkzeug.exceptions import NotFound
+
+class AwardeeApi(BaseApi):
+  def __init__(self):
+    super(AwardeeApi, self).__init__(HPODao(), get_returns_children=True)
+
+  @auth_required(PTC_AND_HEALTHPRO)
+  def get(self, p_id=None):
+    if p_id:
+      hpo = self.dao.get_by_name(p_id)
+      if not hpo:
+        raise NotFound("Awardee with ID %s not found" % p_id)
+      return self._make_response(self.dao.get_with_children(hpo.hpoId))
```

Comment:
Confirming: we don't expect this to be a very large list in the foreseeable future? In terms of total number of sites? And we don't expect any of this metadata to be particularly large? If we do expect it to grow to O(many-thousands) or need large payloads here, then we should consider moving to a more standard REST pattern, e.g.

/awardees
/awardees/{id}/organizations
/awardees/{id}/organizations/{id}/sites

Though we could always change this later if needed as well.

Reply:
The list of awardees and organizations won't be that big. 

The list of sites, OTOH, may grow to be in the thousands (e.g. if every Walgreens location becomes a separate site.)

However, for the time being we're punting on worrying about that. :)